src/components/Security/Roles/Page.vue

```diff
@@ -1,6 +1,20 @@
 <template>
-  <div class="RolesManagement">
-    <headline title="Role Management" />
+  <b-container class="RolesManagement">
+    <b-row>
+      <b-col cols="8">
```

Comment:
I was wondering if when specified in bs4 cols must be equals to 12 ? (wont block for that anyway)

Reply:
We are free to set cols<=12 and its very usefull when we set a property like `justify="center"` to center horizontally the col in the row, wich isn't possible if the col=12.
Also, if there is no other col in the same row, set cols<12 won't impact other cols in other rows.